Source/Diagnostics/BTDiagnostics.H

```diff
@@ -14,47 +14,88 @@ private:
     /** Read relevant parameters for BTD */
     void ReadParameters ();
     /** Flush m_mf_output and particles to file. */
-    // Currently this is an empty function. When OpenPMD/Plotfile is supported
-    // this function can be moved to the base class Diagnostics.
-    void Flush (int /*i_buffer*/) override {}
+    // Currently, a temporary customized output format for the buffer
+    // data is implemented and called in this function.
+    // After implementing OpenPMD and plotfile formats, this function
+    // can be defined and implemented in Diagnostics.H, such that,
+    // the function call to flush out buffer data for
+    // FullDiagnostics and BTDiagnostics is the same
```

Comment:
I think it's alright to put this in the Doxygen doc, but that's just a suggestion
```suggestion
     * Currently, a temporary customized output format for the buffer
     * data is implemented and called in this function.
     * After implementing OpenPMD and plotfile formats, this function
     * can be defined and implemented in Diagnostics.H, such that,
     * the function call to flush out buffer data for
     * FullDiagnostics and BTDiagnostics is the same */
```